README.md

```diff
@@ -1,568 +1,181 @@
-```
-
-### Python
-
-#### `pip3` or `poetry`
+<p align="center">
+    <b>AIConfig - the open-source framework for building production-grade AI applications!</b> <br />
+    Save your prompts, models, and model parameters separately from your application code for improved collaboration, better management, and a faster path from prototype to production.
+</p>
+
+<p align="center">
+  <a href="https://github.com/lastmile-ai/aiconfig/actions/workflows/main_python.yml">
+    <img src="https://github.com/lastmile-ai/aiconfig/actions/workflows/main_python.yml/badge.svg" alt="Python">
+  </a> |
+  <a href="https://github.com/lastmile-ai/aiconfig/actions/workflows/main-typescript.yml">
+    <img src="https://github.com/lastmile-ai/aiconfig/actions/workflows/main-typescript.yml/badge.svg" alt="Node">
+  </a> |
+  <a href="https://github.com/lastmile-ai/aiconfig/actions/workflows/test-deploy-docs.yml">
+    <img src="https://github.com/lastmile-ai/aiconfig/actions/workflows/test-deploy-docs.yml/badge.svg" alt="Docs">
+  </a> |
+  <a href="https://discord.com/invite/xBhNKTetGx">
+    <img src="https://img.shields.io/badge/Discord-LastMile%20AI-Blue?color=rgb(37%2C%20150%2C%20190)" alt="Discord">
+  </a> |
+  <br>
+  <a href="https://aiconfig.lastmileai.dev/">
+    <strong> Documentation</strong>
+  </a>
+</p>
 
 ```bash
+# for python installation:
 pip3 install python-aiconfig
-```
-
-```bash
-poetry add python-aiconfig
-```
-
-[Detailed installation instructions](https://aiconfig.lastmileai.dev/docs/getting-started/#installation).
-
-### Set your OpenAI API Key
-
-> **Note**: Make sure to specify the API keys (such as [`OPENAI_API_KEY`](https://platform.openai.com/api-keys)) in your environment before proceeding.
+# or using poetry: poetry add python-aiconfig
 
-In your CLI, set the environment variable:
-
-```bash
-export OPENAI_API_KEY=my_key
+# for node.js installation:
+npm install aiconfig
+# or using yarn: yarn add aiconfig
 ```
 
-## Getting Started
-
-> We cover Python instructions here, for Node.js please see the [detailed Getting Started guide](https://aiconfig.lastmileai.dev/docs/getting-started)
-
-In this quickstart, you will create a customizable NYC travel itinerary using `aiconfig`.
-
-This AIConfig contains a prompt chain to get a list of travel activities from an LLM and then generate an itinerary in an order specified by the user.
+AIConfigs are JSON-serializable configs that store your prompts, models, and model parameters. Here is a sample AIConfig that uses gpt-3.5-turbo and gpt-4:
 
-> **Link to tutorial code: [here](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Getting-Started)**
-
-https://github.com/lastmile-ai/aiconfig/assets/81494782/805173d1-0f83-44c5-b570-c776bb7dba66
-
-### Download `travel.aiconfig.json`
-
-> **Note**: Don't worry if you don't understand all the pieces of this yet, we'll go over it step by step.
-
-```json
+<details style="border: 1px solid #e8e8e8; padding: 10px; border-radius: 10px;">
```

Comment:
Just validating that you've run this updated aiconfig and it works correctly?

Reply:
yep! ran it in colab and it works